2017-08-04  Jakub Jelinek  <jakub@redhat.com>

	* include/system/sys/ptrace.h: New file.

--- libsanitizer/include/system/sys/ptrace.h.jj	2017-08-04 09:31:11.129824715 +0200
+++ libsanitizer/include/system/sys/ptrace.h	2017-08-04 09:34:36.286450409 +0200
@@ -0,0 +1,7 @@
+#include_next <sys/ptrace.h>
+#ifndef PTRACE_GETREGSET
+/* glibc before
+   https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=cbff0d9689c4d68578b6a4f0a17807232506ea27
+   doesn't define PTRACE_GETREGSET.  */
+#define PTRACE_GETREGSET 0x4204
+#endif
